src/components/TopBar.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import styled from 'styled-components';
+
+export interface TopBarInterface {
+	text: string;
+}
+
+const TopBarStyle = styled.div`
+	display: flex;
+	flex-direction: column;
+	font-weight: 600;
```

Comment:
Just curious.  Why not just use `font-weight: bold`?

Reply:
just changed font-weight to bold after reading your comment, makes way more sense!

Reply:
+1 to that